src/App.tsx

```diff
@@ -1,204 +1,59 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+import { TodoCompletedCategory } from './types/TodoCompletedCategory';
+import { filterTodosByComplated } from './utils/filterTodosByCompleted';
+import { Errors } from './types/Errors';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [completedCategory, setCompletedCategory] =
+    useState<TodoCompletedCategory>(TodoCompletedCategory.all);
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.noneError);
+  const countOfNotCompletedTodos = todos.filter(todo => !todo.completed).length;
+  const filtredTodos = filterTodosByComplated(todos, completedCategory);
+
+  async function setTodosFromApi() {
```

Comment:
will be better.  u load the data and then do a `set`, not just a set
```suggestion
  async function fetchTodosFromApi() {
```